core/src/main/scala/org/apache/spark/deploy/security/HadoopFSDelegationTokenProvider.scala

```diff
@@ -121,6 +121,15 @@ private[deploy] class HadoopFSDelegationTokenProvider
     creds
   }
 
+  private def cancelDelegationTokens(hadoopConf: Configuration,
+                                     creds: Credentials) : Unit = {
+
+    creds.getAllTokens
+      .asScala
+      .filter(_.decodeIdentifier().isInstanceOf[AbstractDelegationTokenIdentifier])
+      .foreach { token => token.cancel(hadoopConf) }
```

Comment:
Trivial, but `.foreach(_.cancel(hadoofConf))` is good too.
Does this even need to be a separate method?